python-sdk/nuscenes/panoptic/panoptic_utils.py

```diff
@@ -0,0 +1,206 @@
+"""
+nuScenes-panoptic utils.
+Code written by Motional and the Robot Learning Lab, University of Freiburg.
+"""
+from typing import Any, Dict, Iterable, List, Tuple
+
+import numpy as np
+from nuscenes.lidarseg.lidarseg_utils import colormap_to_colors, create_lidarseg_legend
+from nuscenes.utils.color_map import get_colormap
+from nuscenes.utils.data_io import load_bin_file
+
+create_panoptic_legend = create_lidarseg_legend
+
+
+def stuff_start_cat_id() -> int:
```

Comment:
Make it a constant variable instead?

Reply:
Or a @property?

Reply:
changed to constant variable, @property is ony for class member.  [0c8dfef](https://github.com/nutonomy/nuscenes-devkit/pull/628/commits/0c8dfef5c8d63f1eb299f704714f0a346be05481)